cli/install/install.go

```diff
@@ -1109,6 +1112,25 @@ func installTarantoolEE(binDir string, includeDir string, installCtx InstallCtx,
 	return nil
 }
 
+// installTarantoolDev creates a symlink to the tarantool executable,
+// which is searched in buildDir.
+func installTarantoolDev(binDir string, buildDir string) error {
+	checkedPaths := make([]string, 0)
+	for _, binaryRelPath := range [...]string{"src/tarantool", "tarantool/src/tarantool"} {
+		binaryPath := filepath.Join(buildDir, binaryRelPath)
+		if util.IsRegularFile(binaryPath) {
+			err := util.CreateSymlink(binaryPath, filepath.Join(binDir, "tarantool"), true)
+			if err == nil {
+				log.Infof("Done.")
```

Comment:
Please make comment more informative. 
Or add a comment above "Searching for tarantool binary...."
"Tarantool binary found, creating symlink..."